static/js/tab_bar.js

```diff
@@ -90,19 +90,12 @@ function bind_title_area_handlers() {
         }
     });
 
-    const color = $(".search_closed").css("color");
-    const night_mode_color = $(".nightmode .closed_icon").css("color");
-
-    // make sure that hover plays nicely with whether search is being
-    // opened or not.
+    // handler that makes sure that hover plays nicely
+    // with whether search is being opened or not.
     $(".narrow_description > a").hover(() => {
-        if (night_mode_color) {
-            $(".search_closed").css("color", night_mode_color);
-        } else {
-            $(".search_closed").css("color", color);
-        }
+        $(".search_closed").addClass("search_icon_hover_highlight");
```

Comment:
Let's make these selectors more specific to include the container; I can easily imagine someone adding a `search_closed`